testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/LogContent.java

```diff
@@ -36,6 +36,8 @@
 import java.util.List;
 import java.util.regex.Pattern;
 
+import static org.codehaus.plexus.util.ReaderFactory.UTF_8;
```

Comment:
```suggestion
import static java.nio.charset.StandardCharsets.UTF_8;
```

Reply:
yes java i expected to be working as well, but it seems not to be. Thanks for the feedback.